src/transformers/generation_tf_utils.py

```diff
@@ -455,7 +456,7 @@ def _generate_no_beam_search(
 
         while cur_len < max_length:
             model_inputs = self.prepare_inputs_for_generation(
-                input_ids, past=past, attention_mask=attention_mask, use_cache=use_cache
+                input_ids, past=past, attention_mask=attention_mask, use_cache=use_cache, **kwargs
```

Comment:
@ratthachat - all the changes you need to do ;-)

Reply:
I saw it 😛 😸   ! Thanks for the refactor Patrick!